tests/js/test-caseholder.js

```diff
@@ -0,0 +1,495 @@
+/* Tests for the "express" and "router" module */
+"use strict";
+var fluid        = fluid || require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+var jqUnit       = require("jqUnit");
+
+fluid.registerNamespace("gpii.express.tests.expressTestCaseHolder");
+
+fluid.setLogging(true);
+
+gpii.express.tests.expressTestCaseHolder.isSaneResponse = function (response, body) {
+
+    jqUnit.assertEquals("The response should have a reasonable status code", 200, response.statusCode);
+    if (response.statusCode !== 200) {
+        console.log(JSON.stringify(body, null, 2));
+    }
+
+    jqUnit.assertValue("There should be a body.", body);
+};
+
+gpii.express.tests.expressTestCaseHolder.assembleUrl = function(baseUrl, path) {
+    var fullPath;
+    // We have to be careful of double slashes (or no slashes)
+    if (baseUrl[baseUrl.length -1] === "/" && path[0] === "/") {
+        fullPath = baseUrl + path.substring(1);
+
+    }
+    else if (baseUrl[baseUrl.length -1] !== "/" && path[0] !== "/") {
+        fullPath = baseUrl + "/" + path;
+    }
+    else {
+        fullPath = baseUrl + path;
+    }
+    return fullPath;
+};
+
+gpii.express.tests.expressTestCaseHolder.verifyStaticRouterModule = function (response, body) {
```

Comment:
Can condense all of these 4 into a single function "verifyContent" which takes the required content as a further (string) argument and uses indexOf rather than regexp.match